airbyte-webapp/src/scss/_variables.scss

```diff
@@ -22,3 +22,5 @@ $width-modal-sm: 492px;
 $width-modal-md: 585px;
 $width-modal-lg: 940px;
 $width-modal-xl: 1008px;
+
+$z-tooltip: 10;
```

Comment:
:tada: for starting to manage all z-indexes globally (so they are not that often mixed up). Could we maybe even put them in their own `z-index`?

Reply:
@timroes Do you mean a `z-index` file?